src/main/java/space/obminyashka/items_exchange/rest/UserController.java

```diff
@@ -165,10 +165,15 @@ public ResponseEntity<String> updateUserPassword(@Valid @RequestBody ChangePassw
     @Operation(summary = "reset user password")
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200", description = "OK"),
-            @ApiResponse(responseCode = "400", description = "BAD REQUEST")
-    })
-    public String resetPassword(@Valid @RequestBody ValidatedEmailRequest validatedEmailRequest){
-        mailService.sendMessagesToEmailForResetPassword(validatedEmailRequest.email());
+            @ApiResponse(responseCode = "400", description = "BAD REQUEST")})
+    @ResponseStatus(HttpStatus.OK)
+    public String resetPassword(@Valid @RequestBody ValidatedEmailRequest validatedEmailRequest,
```

Comment:
It seems I missed that but the name of object is incorrect. 'Validated' in it's name says the value is validated, but instead it needs to be verified other than validated.

Reply:
changed